p11-kit/rpc-client.c

```diff
@@ -788,7 +788,7 @@ rpc_C_Initialize (CK_X_FUNCTION_LIST *self,
 		if (ret != CKR_OK) {
 			assert (module->vtable->disconnect != NULL);
 			(module->vtable->disconnect) (module->vtable, reserved);
-			ret = (module->vtable->connect) (module->vtable, reserved);
+			(module->vtable->connect) (module->vtable, reserved);
```

Comment:
Should we actually check the return value?

Reply:
if the connect wont succeed then the next call fails anyway so I am not sure. maybe.

Reply:
actually, lets check the return value